tldw_Server_API/tests/Characters/test_character_functionality_db.py

```
@@ -81,6 +81,22 @@ def sample_card_data(name="Test Character", **kwargs) -> dict:
     return data
 
 
+def normalize_expected_character_tags(tags: Any) -> set[str]:
```

Ruff (0.15.11)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `tags`

(ANN401)